covalent/_workflow/electron.py

```diff
@@ -276,6 +289,19 @@ def get_item(e, key):
                 iterable_metadata["call_before"] = filtered_call_before
 
                 get_item_electron = Electron(function=get_item, metadata=iterable_metadata)
+
+                # Pack with main electron unless it is a sublattice.
+                name = active_lattice.transport_graph.get_node_value(self.node_id, "name")
+                get_item_electron = (
+                    Electron(function=get_item, metadata=iterable_metadata)
+                    if name.startswith(sublattice_prefix)
+                    else Electron(
+                        function=get_item,
+                        metadata=iterable_metadata,
+                        task_group_id=self.task_group_id,
+                        packing_tasks=True,
```

Comment:
Duplicate code (in the ones below as well) should be extracted into a function instead.